modules/admin_manual/pages/_partials/maintenance/upgrading/upgrade_steps.adoc

```diff
@@ -0,0 +1,63 @@
+= Upgrade PHP to Version {to-version}
+:redhat-software-collections-overview-url: https://developers.redhat.com/products/softwarecollections/overview
+
+To upgrade to PHP {to-version} you first need to subscribe to {redhat-software-collections-overview-url}[the Red Hat Software Collections] channel repository to download and install the PHP {to-version} package in RHEL 7 (if you've not done this already). 
+This documentation uses the same command as you will find there.
+
+[IMPORTANT]
+====
+Ensure that you have `subscription-manager` installed.
+If you don't, yet, have it installed, do so with the following command:
+
+[source,console]
+----
+yum install --assumeyes subscription-manager
+----
+====
+
+== Install the Required Packages
+
+Then, proceed by installing the required PHP {to-version} packages. 
+You can use the command below to save you time.
+
+[source,console,subs="attributes+"]
+----
+yum install \
+  rh-php{to-pkg-version} \
+  rh-php{to-pkg-version}-php \
+  rh-php{to-pkg-version}-php-gd \ 
+  rh-php{to-pkg-version}-php-intl \ 
+  rh-php{to-pkg-version}-php-ldap \
+  rh-php{to-pkg-version}-php-mbstring \ 
+  rh-php{to-pkg-version}-php-mysqlnd 
```

Comment:
hmmm - this list does not seem long enough, but it is a cut-paste from the previous documentation. So I guess it is a separate issue to verify exactly what all the needed extensions are for a successful core install.

Reply:
I forgot to add a note yesterday, for someone else with a RHEL system **and** a paid subscription, to check if these are all the packages that are required. 

I spent a good chunk of time yesterday, trying unsuccessfully to get my free subscription working - _I'm downright positive that I was able to do so a year or two ago._ In the end, I couldn't and felt that I was just wasting time continuing to hack away at it, and that asking someone with a working system might just be the more logical solution.

Reply:
In the php ansible role we install the following packages on RHEL:
```
rh-php{to-pkg-version} \
rh-php{to-pkg-version}-php \
rh-php{to-pkg-version}-php-cli \
rh-php{to-pkg-version}-php-common \
rh-php{to-pkg-version}-php-devel \
rh-php{to-pkg-version}-php-fpm \
rh-php{to-pkg-version}-php-gd \
rh-php{to-pkg-version}-php-ldap \
rh-php{to-pkg-version}-php-mbstring \
rh-php{to-pkg-version}-php-opcache \
rh-php{to-pkg-version}-php-mysqlnd \
rh-php{to-pkg-version}-php-pdo \
rh-php{to-pkg-version}-php-pear \
rh-php{to-pkg-version}-php-xml \
rh-php{to-pkg-version}-php-xmlrpc \
rh-php{to-pkg-version}-php-zip \
rh-php{to-pkg-version}-php-intl \
rh-php{to-pkg-version}-php-curl
```
https://github.com/owncloud-ansible/php/blob/e090faa74bd4929399bdd8770e43ee9f2a256e37/vars/redhat.yml#L2

Reply:
Actually I can confirm this list includes the very minimum requirements in terms of PHP packages needed for a basic installation, that is basic PHP extensions for the installation to be successfully completed. Just tested this on RHEL 7.6.

Reply:
Thanks @pako81 @enbrnz. Would it be worth including the full list? I'm fine with updating the list.

Reply:
@settermjd These are the minimum requirements for a working oC installation: 

```
[root@localhost ~]# yum list installed | grep php
rh-php70.x86_64                2.3-1.el7               @rhel-server-rhscl-7-rpms
rh-php70-php.x86_64            7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-cli.x86_64        7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-common.x86_64     7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-gd.x86_64         7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-intl.x86_64       7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-json.x86_64       7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-ldap.x86_64       7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-mbstring.x86_64   7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-mysqlnd.x86_64    7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-pdo.x86_64        7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-pear.noarch       1:1.10.1-3.el7          @rhel-server-rhscl-7-rpms
rh-php70-php-process.x86_64    7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-xml.x86_64        7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-php-zip.x86_64        7.0.27-2.el7            @rhel-server-rhscl-7-rpms
rh-php70-runtime.x86_64        2.3-1.el7               @rhel-server-rhscl-7-rpms
```

Compared to https://github.com/owncloud/docs/pull/2172#discussion_r362455834 we are missing `rh-php{to-pkg-version}-php-devel`, `rh-php{to-pkg-version}-php-xmlrpc` and `rh-php{to-pkg-version}-php-curl` which I don't think are strictly required. While `rh-php{to-pkg-version}-php-opcache` even if not strictly required is strongly recommended.

The question is also if we want to differentiate between what is needed for oC to run and what is also recommended/suggested. See https://doc.owncloud.com/server/admin_manual/installation/server_prep_ubuntu_18.04.html#common-prerequisites for how we do it for Ubuntu 18.04.

My two cents: list only the minimum requirements for everything different from Ubuntu. That is, the initially documented list plus eventually `rh-php{to-pkg-version}-php-opcache`.

Maybe @enbrnz can add his thoughts here.

Reply:
> My two cents: list only the minimum requirements for everything different from Ubuntu. That is, the initially documented list plus eventually `rh-php{to-pkg-version}-php-opcache`.

Agreed, php-devel & php-pear are being installed in order to support Redis on RHEL/CentOS. php-fpm is entirely unnecessary.

I am surprised php-ldap is required, even in a minimal installation, everything else sounds fine (including opcache).

Reply:
@enbrnz thx. yeah, actually php-ldap is not strictly required but I guess we are listing it for staying aligned with Ubuntu requirements where it is listed as required. Fine to me to not include it.

Reply:
Thanks for the continued discussion. I'll get the list updated today.

Reply:
The list has been updated.